docker-compose-watcher.yml

```
@@ -15,39 +15,39 @@ services:
       timeout: 3s
       retries: 5
 
-  watcher:	
+  watcher:
     #last stable integration watcher	
-    image: omisego/watcher:4445aee	
-    command: "full_local"	
-    environment:	
-      - ETHEREUM_RPC_URL=https://ropsten.infura.io/v3/${INFURA_API_KEY}	
-      - CHILD_CHAIN_URL=https://childchain.ropsten.v1.omg.network	
-      - ETHEREUM_NETWORK=ROPSTEN	
-      - AUTHORITY_ADDRESS=0x3272b97b7f1b74b338cb0fdda167cf76bc4da3b6	
-      - TXHASH_CONTRACT=0x25e445594f425a7a94141a20b8831580953b92ddd0d12e9c775c571e4f3da08c	
-      - CONTRACT_ADDRESS_PLASMA_FRAMEWORK=0xa72c9dceeef26c9d103d55c53d411c36f5cdf7ec	
-      - CONTRACT_ADDRESS_ETH_VAULT=0x2c7533f76567241341d1c27f0f239a20b6115714	
-      - CONTRACT_ADDRESS_ERC20_VAULT=0x2bed2ff4ee93a208edbf4185c7813103d8c4ab7f	
-      - CONTRACT_ADDRESS_PAYMENT_EXIT_GAME=0x960ca6b9faa85118ba6badbe0097b1afd8827fac	
-      - DATABASE_URL=postgres://omisego_dev:omisego_dev@postgres:5432/omisego_dev	
-      - PORT=7434	
-      - DD_DISABLED=true	
-      - DB_PATH=/app/.omg/data	
-      - ETHEREUM_EVENTS_CHECK_INTERVAL_MS=8000	
-      - ETHEREUM_STALLED_SYNC_THRESHOLD_MS=300000	
-      - ETHEREUM_BLOCK_TIME_SECONDS=15	
-      - EXIT_PROCESSOR_SLA_MARGIN=5520	
+    image: omisego/watcher:4445aee
```

Not to do with your PR, but if I could sneak this latest version in (should already be compatible with `*.ropsten.v1.omg.network`

```suggestion
    image: omisego/watcher:v0.4.8
```

Also applies to watcher_info below

---

Hmmm. I'm doing this from master branch. Maybe it's time to update the version there?